lib/api/dsl/storage/index.js

```diff
@@ -200,16 +222,11 @@ class Storage {
    * @return {Promise<*>}
    */
   remove (filterId) {
-    let
-      index,
-      collection;
-
     if (!this.filters[filterId]) {
-      return Promise.reject(new NotFoundError(`Unable to remove filter "${filterId}": filter not found`));
+      return Bluebird.reject(new NotFoundError(`Unable to remove filter "${filterId}": filter not found`));
     }
 
-    index = this.filters[filterId].index;
-    collection = this.filters[filterId].collection;
+    const {index, collection} = this.filters[filterId];
 
     removeFromTestTables(this.filters, this.subfilters, this.testTables, index, collection, this.filters[filterId]);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 116 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)